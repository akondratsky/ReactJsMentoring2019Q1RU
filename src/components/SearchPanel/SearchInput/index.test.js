import React from 'react';
import { SearchInput } from './index';

describe('<SearchInput />', () => {
  test('should be rendered correctly', () => {
    const input = shallow(
      <SearchInput />
    );
    expect(input).toMatchSnapshot();
  });
});