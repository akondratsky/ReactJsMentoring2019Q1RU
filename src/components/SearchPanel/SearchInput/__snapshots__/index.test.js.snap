// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SearchInput /> should be rendered correctly 1`] = `
<div
  className="search-input"
>
  <div
    className="search-input__input"
  >
    <DummyInput />
  </div>
  <div
    className="search-input__controls"
  >
    <span
      className="search-input__text"
    >
      Search by
    </span>
    <div
      className="search-input__switcher"
    >
      <Switcher
        default="title"
        onChange={[Function]}
        variants={
          Array [
            "title",
            "genre",
          ]
        }
      />
    </div>
    <div
      className="search-input__submit"
    >
      <DummyButton
        isActive={true}
        text="Search"
        width="100px"
      />
    </div>
  </div>
</div>
`;
